lib/beaker-pe/install/pe_utils.rb

```diff
@@ -365,6 +367,92 @@ def deploy_frictionless_to_master(host)
         # @api private
         #
         def do_install hosts, opts = {}
+          # detect the kind of install we're doing
+          install_type = determine_install_type(hosts, opts)
+          case install_type
+          when :simple_monolithic
+            simple_monolithic_install(hosts.first, hosts.drop(1), opts)
```

Comment:
I think it's safe to assume that because it's currently assumed throughout the whole file. If that were to stop being true, many other things would break.